### ISSUE TITLE
spec.yaml or README.md out of date

### ISSUE THREAD

**Issue by kmilodenisglez** [hyperledger-firefly/fabconnect#103]
Hi @nguyer, @jimthematrix

The **"payloadType"** property of the **"SubscriptionInfo"** structure in _internal/events/api/event.go_ accepts the data types "bytes", "string" or "stringifiedJSON/json". 

or is the README.md the one that is not updated?

But in the **spec.yaml** only "json" and "string" are accepted.

![image](https://user-images.githubusercontent.com/24257221/217869062-c62eaf27-4cbe-49ce-841a-60efd74aab32.png)

**Comment by nguyer** [comment]
I think this is correct. In this case "bytes" are passed in as a string encded as hex or base64.

**Comment by kmilodenisglez** [comment]
> I think this is correct. In this case "bytes" are passed in as a string encded as hex or base64.

so you have to update the README.md, right?

Because in the readme they use the value "stringifiedJSON" for the "payloadType" property, at https://github.com/hyperledger/firefly-fabconnect#json-data-support-in-events

**Comment by nguyer** [comment]
Sorry I'm not understanding or seeing anything that is not correct here. "Stringified JSON" is just a string so it's still valid.

**Comment by kmilodenisglez** [comment]
> Sorry I'm not understanding or seeing anything that is not correct here. "Stringified JSON" is just a string so it's still valid.

When I pass the string "payloadType" as a parameter, this is the response returned by the endpoint "http://localhost:3000/subscriptions":

![image](https://user-images.githubusercontent.com/24257221/217959754-9c4643bc-94c5-4b14-8ec7-52f86e6e5302.png)


![image](https://user-images.githubusercontent.com/24257221/217959312-c4d8e0ec-c68a-4767-93ce-a32477815ba5.png)

**Comment by kmilodenisglez** [comment]
It only lets me pass the values "string" or "json"

**Comment by nguyer** [comment]
Ah okay, I get it. Sorry I was slow to understand this one! I think the readme can be changed to just say "string" there instead.

**Comment by kmilodenisglez** [comment]
> Ah okay, I get it. Sorry I was slow to understand this one! I think the readme can be changed to just say "string" there instead.

Exactly, sorry if I didn't explain well. My English is not very good

**Comment by kmilodenisglez** [comment]
pull-request: https://github.com/hyperledger/firefly-fabconnect/pull/104
